js/src/personal/riyuan/components/footer/Footer.tsx

```diff
@@ -0,0 +1,76 @@
+import { Anchor, Group, ActionIcon, rem } from '@mantine/core'
+import {
+  IconBrandTwitter,
+  IconBrandYoutube,
+  IconBrandInstagram,
+} from '@tabler/icons-react'
+// @ts-ignore
+import classes from './FooterCentered.module.css'
+
+const links = [
+  { link: '#', label: 'Contact' },
+  { link: '#', label: 'Privacy' },
+  { link: '#', label: 'Blog' },
+  { link: '#', label: 'Store' },
+  { link: '#', label: 'Careers' },
+]
+
+export function FooterCentered() {
+  const items = links.map((link) => (
```

Comment:
nit: `items` and `links` are a bit vague especially when there are also hard coded external links to github and such later on.

Reply:
change items to social